rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -1244,3 +1253,26 @@ void memory_viewer_panel::ShowImage(QWidget* parent, u32 addr, color_format form
 		f_image_viewer->setFixedSize(f_image_viewer->sizeHint());
 	});
 }
+
+void memory_viewer_panel::ShowAtPC(u32 pc)
+{
+	if (Emu.IsStopped())
+		return;
+
+	if (const auto* fxo = g_fxo->try_get<memory_viewer_fxo>())
```

Comment:
I feel like this is overkill. Could probably just store the pointer as a static member and set it to nullptr if the emu id changed.

Reply:
> I feel like this is overkill. Could probably just store the pointer as a static member and set it to nullptr if the emu id changed.

I had the same feeling when implementing this. If it needs to be changed, let me know :+1: